doc/api/fs.md

```diff
@@ -3554,6 +3554,39 @@ It is possible to configure the console to render UTF-8 properly by changing the
 active codepage with the `chcp 65001` command. See the [chcp][] docs for more
 details.
 
+## fs.writev(fd, buffers, [position], callback);
+<!-- YAML
+added: REPLACEME
+-->
+
+* `fd` {integer}
+* `buffers` {ArrayBufferView[]}
+* `position` {integer}
+* `callback` {Function}
+  * `err` {Error}
+  * `bytesWritten` {integer}
+  * `buffers` {ArrayBufferView[]}
+
+Write an array of `ArrayBufferView` to the file specified by `fd` using `writev`.
+
+`position` refers to the offset from the beginning of the file where this data
```

Comment:
does ArrayBufferView increment iov base addresses for us under the hood?

using a file position param feels a little confusing to me because
it raises the question immediately in my mind:
* aren't we missing all the iovec address positions?

is `position` a single number or does it correspond to the buffers?

```man
from WRITE(2)       BSD System Calls Manual:
```

For writev(), the iovec structure is defined as:
```c
struct iovec {
  char   *iov_base;  /* Base address. */
  size_t iov_len;    /* Length. */
};
```
> Each iovec entry specifies:
> * the base address and
> * length of an area in memory

> from which data should be written.